Gemfile

```diff
@@ -1,2 +1,2 @@
-source 'https://rubygems.org'
+source "https://rubygems.org"
```

Comment:
Yes, I would prefer single quotes. But, please, don't start a discussion on this topic again. Just tell me what to use...